Zlib.autopkg

```diff
@@ -25,7 +25,7 @@ nuget
          Breaking changes:
          Bug fixes:
          Misc:
-            - Updated toolset to v141 for Visual Studio 2017
+            - Updated toolset to v142 for Visual Studio 2019
          ";
       copyright: @"Copyright (c) 2018.";
       tags: { zlib, CoApp, compression, vs2017, native };
```

Comment:
Looking at when we previously made updates to visual studio , Should also add vs2019 to the tags

Reply:
Done! Thanks.